iphone/Classes/TiUIListItem.m

```diff
@@ -252,6 +252,8 @@ - (void)setPosition:(int)position isGrouped:(BOOL)grouped
 
 - (void)applyBackgroundWithSelectedColor:(id)selectedBackgroundColor selectedImage:(id)selectedBackgroundImage
 {
+  if (!selectedBackgroundColor && !selectedBackgroundImage) return; // Ignore custom selection styles for native selections
```

Comment:
If one of those properties is set, a selection view with that background color will be created. In the past, Titanium had hardcoded values for the native selection styles, causing huge issues on dark mode, since there are not semantic. With this check, we make sure to not override the native styles anymore, if no custom selection styles are used.